src/native_module/core.cljs

```diff
@@ -11,6 +11,41 @@
   (when (exists? (.-NativeModules react-native))
     (.-Status ^js (.-NativeModules react-native))))
 
+(defn account-manager
+  []
+  (when (exists? (.-NativeModules react-native))
+    (.-AccountManager ^js (.-NativeModules react-native))))
```

Comment:
Why do we need to make `account-manager` and its friends functions? Couldn't we define them as static values? It seems we just grab static properties from `react-native`.

Or is this a performance trick to reduce memory consumption at startup?

Reply:
I'm not really sure of the perf impacts of making it a `defn` vs `def`.

To be honest I simply followed the existing pattern and the way things were accessed was by grabbing these properties from `NativeModules` and storing them in functions.

I did try your suggestion of making them static and found no issues so I guess we can keep them as static for now and update in a separate PR if we find any performance improvements for moving back to functions.

addressed here : https://github.com/status-im/status-mobile/pull/18385/commits/4fdb0d80235a6a519e86f978c6a4d1ed50dc85a5

Reply:
@ilmotta : Unfortunately this approach causes a crash in release `apk` , hence I had to revert back to using functions here https://github.com/status-im/status-mobile/pull/18385/commits/2086ec70474299eb5bc5017d9558ad6b30e33e03

Reply:
Quite interesting @siddarthkay. Thanks for attempting this refactor. I'm curious about the error, but let's not waste time on that right?

Reply:
It was a cryptic message about unable to load modules on `undefined` I found it via logcat which helped me figure out that the root cause was this commit :)